cloudhub-2/modules/ROOT/pages/ps-config-app-level-egress.adoc

```diff
@@ -0,0 +1,165 @@
+
+.  Select *More actions* (image:more-icon.png[More actions menu, 1%,1%]) > *Delete*. 
+. When prompted, click *Delete Rule Group* to confirm the deletion.
++
+[NOTE]
+Once you delete a custom rule group, any applications that use it immediately start using the default egress rule group instead.
+
+image::ps-egress-delete-ruleset.png[Delete custom egress rule group]
+
+You can disable the application-level egress rule feature by clicking the feature's toggle. When prompted, click *Delete and Disable App-Level Rules* to confirm. 
+
+[NOTE]
+Once you disable the application-level egress rule feature, all custom application-level egress rule groups are deleted, and all application-level egress is allowed for all applications.
+
+== Apply Application-Level Egress Rules When Deploying Your Applications
+
+You can apply the application-level egress rules that you configured in your private space when you deploy an application:
+
+. When deploying an application to your private space, select the *Firewall Rules* tab.
++
+. In the *Egress Rule Group* section, select the ruleset to apply to your application. 
+. Click *Apply Changes*.
+
+
+== Configure an Application-Level Egress Connection to Anypoint MQ
+
+To allow HTTPS connections to Anypoint MQ, add an application-level egress rule group with the following rules:
+
+Allow egress to the Anypoint control plane:
+
+* *Protocol*: `HTTPS`
+* *Destination*: `Custom Domains`
+** With value:
+*** For prod: `anypoint.mulesoft.com`
+*** For prod-eu: `eu1.anypoint.mulesoft.com`
+
+Allow egress to the MQ destination:
+
+* *Protocol*: HTTPS
+* *Destination*: `Custom Domains`
+** With the value of the domain name configured in the MQ connector URL
++
+For example, if the MQ connector URL is `https://mq-us-east-1.anypoint.mulesoft.com/api/v1/organizations/{{anypointOrgId}}/environments/{{anypointEnvId}}/`,
+use the value `mq-us-east-1.anypoint.mulesoft.com`.
+
+== Configure an Application-Level Egress Connection to Object Store v2
+
+To allow HTTPS connections to Object Store v2, add an application-level egress rule group with the following rules:
+
+Allow egress to the Anypoint control plane:
+
+* *Protocol*: `HTTPS`
+* *Destination*: `Custom Domains`
+** With value:
+*** For prod: `anypoint.mulesoft.com`
+*** For prod-eu: `eu1.anypoint.mulesoft.com`
+
+Allow egress to the Object Store v2 destination:
+
+* *Protocol*: `HTTPS`
+* *Destination*: `Custom Domains`
+** With value:
+*** For prod: `object-store-us-east-1.anypoint.mulesoft.com`
+*** For prod-eu: `[xxx]`
++
+This URL varies depending on the region. For example, to access Object Store in `ap-southeast-1`, add the URL `object-store-ap-southeast-1.anypoint.mulesoft.com` as the value.
+
+If you enable egress for a domain that includes redirection, such as when `https://salesforce.com` redirects to `https://www.salesforce.com`, allow egress for both the original target, `salesforce.com`, and the redirected target, `www.salesforce.com`.
```

Comment:
We should remove "http://" and "https://" - we are only dealing with domains here, not URLs

Reply:
Also, this statement is only true for the (probably small) use cases where www.salesforce.com resolves to a different IP than salesforce.com